application/modules/admin/views/aclmanager/index.tpl

```diff
@@ -1,24 +1,27 @@
 <section class="box big">
 
 	<h2>
-		What would you like to manage?
+		<img src="{$url}application/themes/admin/images/icons/black16x16/ic_settings.png"/>
+		What do you want to manage?
 	</h2>
 
-<div class="content">
 	<span style="padding-top:22px;padding-bottom:22px;">
 		<a href="{$url}admin/aclmanager/groups" class="big_icon_button" data-tip="Groups are a set of roles that is assigned to users">
+			<img src="{$url}application/modules/admin/images/id.png" style="margin-top:-12px;">
```

Comment:
This, and the following, `margin-top` statements feel a bit random, especially since they're all slightly different. Why is this? Would it make more send to make the image sizes the same as opposed to having separate margins, or are these images used in too many places?

Reply:
without that statement image will "fall" down below Groups text like in my screenshoot below , anyway this have been already merged on srp6 without problems.

![user](https://user-images.githubusercontent.com/24683355/113721672-66760b00-96f8-11eb-8c90-3468a60ceb1f.JPG)

Reply:
I'm not saying it's a problem and it won't work, I'm just saying there might be a better way as opposed to having 3 different (magic) margins used

Reply:
Those values was taken from fusioncms code you can [check this url](https://github.com/Desch/FusionCMS/blob/master/application/modules/admin/views/aclmanager/index.tpl)

Reply:
feel free to update them as you wish

Reply:
Alright 👍🏼